pkgs/development/python-modules/wtforms-sqlalchemy/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  sqlalchemy,
+  wtforms,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "wtforms-sqlalchemy";
+  version = "0.4.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "wtforms";
+    repo = "wtforms-sqlalchemy";
+    rev = version;
+    hash = "sha256-uR09LYfcyre+AC2TTEIhpjSI7y4Yo0GJ20smkzo5PRY=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    sqlalchemy
+    wtforms
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "wtforms_sqlalchemy" ];
+
+  meta = with lib; {
+    description = "WTForms integration for SQLAlchemy";
+    homepage = "https://github.com/wtforms/wtforms-sqlalchemy";
+    changelog = "https://github.com/wtforms/wtforms-sqlalchemy/blob/${src.rev}/CHANGES.rst";
```

Comment:
```suggestion
    changelog = "https://github.com/wtforms/wtforms-sqlalchemy/blob/${lib.removePrefix "refs/tags/" src.rev}/CHANGES.rst";
```

Reply:
we can just use version here, that is a lot easier